web-packages/common/package.json

```diff
@@ -4,46 +4,30 @@
     "version": "0.0.0",
     "type": "module",
     "scripts": {
-        "build_wasm": "bash ../reti-js/init_test.sh",
+        "build_wasm": "bash ../../reti-js/init_test.sh",
```

Comment:
Would there be a way to like run from project root or smth?

Reply:
I don't know, probably not since it's a normal shell script. Unless pnpm has a way to execute scripts from the workspace root instead of project

Reply:
I looked at it and if it is just a script in the root package.json file it should run from there even when called above